prime-router/metadata/fhir_mapping/hl7/datatypes/CQ/Quantity.yml

```diff
@@ -0,0 +1,32 @@
+value:
+  type: BIG_DECIMAL
+  expressionType: HL7Spec
+  valueOf: CQ.1
+
+unit:
+  expressionType: nested
+  vars:
+    cwe_originalText: STRING, CQ.2.9
+  expressions:
+    - condition: $cwe_originalText NULL
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: CQ.2.1
```

Comment:
The mapping inventory says this should be CWE.2 but this seems to point to CWE.1. Was this intended?

Reply:
Good catch!